packages/nimble-components/src/utilities/style/user-select.ts

```diff
@@ -0,0 +1,6 @@
+import { cssPartial } from '@microsoft/fast-element';
+
+export const userSelectNone = cssPartial`
```

Comment:
```suggestion
/**
 * Set user-select: none in a way that works across all supported browsers.
 * https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#browser_compatibility
 */
export const userSelectNone = cssPartial`
```

Reply:
Ah, good catch. I had meant to create a comment here and then it completely slipped my mind.